Gemfile

```diff
@@ -41,6 +41,12 @@ group :development, :test do
   gem 'capybara', '>= 2.15'
   gem 'selenium-webdriver'
   gem 'rubocop', '~> 0.59.2', require: false
+  #TEST
+  gem 'rspec-rails', '~> 3.8'
```

Comment:
As you said, this gems are only  for the testing environment, you can move it  to the :test group

Reply:
Done